src/patternfly/components/Masthead/masthead.scss

```diff
@@ -172,16 +186,16 @@ $pf-v6-c-masthead--inset-map: build-spacer-map("none", "sm", "md", "lg", "xl", "
   grid-column: var(--#{$masthead}__content--GridColumn);
   grid-column-end: -1; // force content to right edge or full available width
   row-gap: var(--#{$masthead}__content-RowGap, var(--#{$masthead}--RowGap));
-  column-gap: var(--#{$masthead}__content-ColumnGap, var(--#{$masthead}--ColumnGap));
+  column-gap: var(--#{$masthead}__content--ColumnGap, var(--#{$masthead}--ColumnGap));
   align-items: var(--#{$masthead}__content--AlignItems, var(--#{$masthead}--AlignItems));
   order: var(--#{$masthead}__content--Order);
 }
 
-// - Masthead brand
-.#{$masthead}__brand {
-  align-items: var(--#{$masthead}__brand--AlignItems, var(--#{$masthead}--AlignItems));
-  max-height: var(--#{$masthead}__brand--MaxHeight);
-  margin-inline-end: var(--#{$masthead}__brand--MarginInlineEnd);
+// - Masthead logo
+.#{$masthead}__logo {
+  align-items: var(--#{$masthead}__logo--AlignItems, var(--#{$masthead}--AlignItems));
+  max-height: var(--#{$masthead}__logo--MaxHeight);
+  margin-inline-end: var(--#{$masthead}__logo--MarginInlineEnd);
```

Comment:
Do we need this margin?

Reply:
I think we need space there so if there's something left aligned in the content area it doesn't butt against - but I will move it to `masthead__main`
<img width="550" alt="image" src="https://github.com/user-attachments/assets/f30b753a-7195-464b-b9b1-4babb6a449ee">

Reply:
Nice, that's a good place for it 👍